### PR TITLE
add 0.26.1

### PR DESCRIPTION
**Pull request by aliotta**

### CODE REVIEW

**Review by neel-astro — APPROVED** [review]

File: Formula/astro.rb [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
   desc "To build and run Airflow DAGs locally via docker-compose and deploy DAGs to Astronomer-managed Airflow clusters and interact with the Astronomer API."
   homepage "https://astronomer.io"
-  version "0.26.0"
+  version "0.26.1"
 
   on_macos do
     if Hardware::CPU.intel?
-      url "https://github.com/astronomer/astro-cli/releases/download/v0.26.0/astro_0.26.0_darwin_amd64.tar.gz"
+      url "https://github.com/astronomer/astro-cli/releases/download/v0.26.1/astro_0.26.1_darwin_amd64.tar.gz"
       sha256 "e978c6deb2d02d32aeff97571041ea1ca9c5b1cd547c052d27c5698195fe7f6d"
```
Comment (andriisoldatenko) [inline Formula/astro.rb:13]: What about this line?

**Review by andriisoldatenko — CHANGES_REQUESTED** [review]

**Review by danielhoherd — APPROVED** [review]
🍺LGTM

File: Formula/astro.rb (hunk 1 above)
Comment (aliotta) [reply Formula/astro.rb:13]: ah good call. Thanks!

Comment (danielhoherd) [inline Formula/astro.rb:13]:
```suggestion
      sha256 "7bf7d9d911bcf9ed462775748f0de4fee0904c8ab66d42591081b06d0e30401c"
```

**Review by danielhoherd — CHANGES_REQUESTED** [review]
Actually we need the sha updated (see comment with suggestion)

File: Formula/astro@0.26.1.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+class AstroAT0260 < Formula
```
Comment (andriisoldatenko) [inline Formula/astro@0.26.1.rb:5]: hm... do you want to backup 0260?

Comment (andriisoldatenko) [inline Formula/astro@0.26.1.rb:5]:
```suggestion
class AstroAT0261 < Formula
```

Comment (danielhoherd) [reply Formula/astro@0.26.1.rb:5]: @andriisoldatenko do you mean 0261 ?

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**